ro_help/hub/views.py

```diff
@@ -183,7 +179,11 @@ def search(self, queryset):
 
     def get_queryset(self):
         needs = self.search(self.get_needs())
-        return needs.filter(**{name: self.request.GET[name] for name in self.allow_filters if name in self.request.GET})
+        filters = {name: self.request.GET[name] for name in self.allow_filters if name in self.request.GET}
+
+        if "tag" in self.request.GET:
```

Comment:
I'm happy to make it nicer if you guys have any ideas.

My first attempt used a special mapping between `request.GET` query params and model fields, however, considering that all the other filters are already covered by the initial abstraction, I decided to handle `tag` as a special one. Mainly because I need to use `.getlist` here

Reply:
It's good, no worries. I would just use `self.request.GET.getlist("tag", "")` instead of `"tag" in self.request.GET` since we encounter `?tag=` as well.